pcsx2/GS/GSCapture.cpp

```diff
@@ -289,13 +289,15 @@ bool GSCapture::LoadFFmpeg(bool report_errors)
 
 	if (report_errors)
 	{
-		Host::ReportErrorAsync(TRANSLATE_SV("GSCapture","Failed to load FFmpeg"),
-			fmt::format(TRANSLATE_FS("GSCapture","You may be missing one or more files, or are using the incorrect version. This build of PCSX2 requires:\n"
-						"  libavcodec: {}\n"
-						"  libavformat: {}\n"
-						"  libavutil: {}\n"
-						"  libswscale: {}\n"
-				"  libswresample: {}\n"), LIBAVCODEC_VERSION_MAJOR, LIBAVFORMAT_VERSION_MAJOR, LIBAVUTIL_VERSION_MAJOR,
+		Host::ReportErrorAsync(TRANSLATE_SV("GSCapture", "Failed to load FFmpeg"),
+			fmt::format(TRANSLATE_FS("GSCapture", "You may be missing one or more files, or are using the incorrect version. This build of PCSX2 requires:\n"
+				"  libavcodec: {}\n"
+				"  libavformat: {}\n"
+				"  libavutil: {}\n"
+				"  libswscale: {}\n"
+				"  libswresample: {}\n\n"
+				"Please refer to our official documentations for more information."),
```

Comment:
"Please refer to our official documentation for more information"  , but otherwise look good

Reply:
Done